vendor/assets/javascripts/behaviors/changed_form_modal.js

```diff
@@ -3,6 +3,9 @@ var CFM = Cmsify.ChangedFormModal = function(el) {
   this.$el = $('form');
   if(!this.$el.length) return;
   this.lastData = this.serializeFields();
+  this.detectChange = this.detectChange.bind(this);
+  this.$el.on('keyup', this.detectChange);
+  this.$el.on('click', this.detectChange);
```

Comment:
Is it possible for you to somehow detect the `sortupdate` event here so it's uniform?